include/timer/timer.h

```diff
@@ -18,28 +21,23 @@ handle more than 35 minutes (which is the same as timer_8bit).
 #ifndef F_CPU
 #define F_CPU 8000000
 #endif
-
-#define PRESCALER 1024.0
-#define T ((PRESCALER / F_CPU) * 1000.0)
-// timer clock period (ms)
-#define MAX_TIME_16BIT (T * 0xFFFF)
-// the maximum time the 16 bit timer can hold until it overflows (ms)
-#define MAX_TIME_8BIT (T * 0xFF)
-// the maximum time the 8 bit timer can hold until it overflows (ms)
-
+#define PRESCALER 1024. //prescaler factor - divide clock by 1024
+#define PERIOD ((PRESCALER / F_CPU) * 1000.0) // timer clock period (ms)
+#define MAX_TIME_16BIT (T * 0xFF) //counter maximum (16-bit) p.127
+#define MAX_TIME_8BIT (T * 0xF) //counter maximum (8 bit) p.127
```

Comment:
I think these are actually supposed to be what they were originally (see p. 132 and 141)